src/PluginRegistrationDialog.cpp

```diff
@@ -918,151 +327,59 @@ void PluginRegistrationDialog::OnRescan(wxCommandEvent& WXUNUSED(evt))
       pm.Save();
       pm.NotifyPluginsChanged();
 
-      if (!failedPlugins.empty())
+      if(self)
       {
-         auto dialog = safenew IncompatiblePluginsDialog(this, wxID_ANY, ScanType::Manual, failedPlugins);
-         dialog->ShowModal();
+         self->ReloadModel();
+         if (!failedPlugins.empty())
+         {
+            auto dialog = safenew IncompatiblePluginsDialog(self.get(), wxID_ANY, ScanType::Manual, failedPlugins);
+            dialog->ShowModal();
```

Comment:
I noticed (and think I remember someone else mentioning this in the ticket, maybe Peter) that once this modal shows, the visible plugin items become greyed out:
![image](https://github.com/audacity/audacity/assets/22740106/0b27817c-0405-434e-b926-c0b16900a18b)

I don't know why or how to fix this, but it probably would be around those lines.

Reply:
> I noticed (and think I remember someone else mentioning this in the ticket, maybe Peter) that once this modal shows, the visible plugin items become greyed out

@saintmatthieu 

Yes I recall that was me - but:
a) I can't find where I reported it,
b) I can't seem to be able to replicate it on the latest branch build for this.

Reply:
@petersampsonaudacity As Matthieu accurately noticed, it seem that rows become grayed out when "Incompatible Plugins" dialog appear. In latest update I provided a hotfix which works for me.
Also, search when typing should now loop over the table.